File: src/unity/Runtime/Ads/IBannerAd.cs
namespace EE {
    public interface IBannerAd : IAd {
        /// <summary>
        /// Gets or set the visibility of this ad.
        /// </summary>
        bool IsVisible { get; set; }

        /// <summary>
        /// Gets or sets the anchor of this ad.
        /// </summary>
        (float, float) Anchor { get; set; }

        /// <summary>
        /// Gets or set the position of this ad.
        /// </summary>
        (float, float) Position { get; set; }

        /// <summary>
        /// Gets or set the size of this ad.
        /// </summary>
        (float, float) Size { get; set; }
    }
}

File: src/ios/ee/ads/IBannerAd.swift
//
//  IAdView.swift
//  ee-x-366f64ba
//
//  Created by eps on 6/24/20.
//

import Foundation

protocol IBannerAd: IAd {
    var isVisible: Bool { get set }
    var position: CGPoint { get set }
    var size: CGSize { get set }
}
